tests/test_persons.py

```diff
@@ -29,3 +27,58 @@ def test_fsearch_api(client, mock_db):
     total, persons = fsearch(last_name=['Einstein'], db=mock_db)
     assert total == 1
     assert persons[0]['tree_version'] == 1
+
+def test_clean_person(mock_db):
+    # two cases for cleaning up the personal info
+    cleaned = clean_person({
+            'name_lc': ['yossi', 'cohen'],
+            'deceased': False,
+            'id': 'I1',
+            'Slug': {'En': 'person_1;0.I7'},
+            'bio': 'yossi is a big boy'
+        })
+    assert 'bio' not in cleaned
+
+    cleaned = clean_person({
+            'name_lc': ['yossi', 'cohen'],
+            'birth_year': 2000,
+            'id': 'I1',
+            'Slug': {'En': 'person_1;0.I7'},
+            'bio': 'yossi is a big boy'
+        })
+    assert 'bio' not in cleaned # this will FAIL in 2100
```

Comment:
can use [freezegun](https://github.com/spulec/freezegun) to travel through time..